app/helpers/patient_query_helper.rb

```diff
@@ -491,6 +464,50 @@ def advanced_filter(patients, filters, tz_offset)
   end
   # rubocop:enable Metrics/MethodLength
 
+  def advanced_filter_age(patients, filter)
+    # when the drop down option is blank, the filter should only return patients without a date_of_birth
+    return patients.where(date_of_birth: nil) if filter[:numberOption].blank?
```

Comment:
Should this be searching on a nil age field or a nil DOB field?

Reply:
also should this check for `''` too?

Reply:
`blank?` checks for `nil` and `''`